### ISSUE TITLE
Voice Attack "Environment" does not update to "Normal space" on Planetary Landing

### ISSUE THREAD

**Issue by Corysia** [EDCD/EDDI#709]
## EDDI version in which issue found
EDDI 3.0.1 beta 4

## Steps to reproduce

Planetary landing.

## Expected
Environment to contain "Normal space"

## Observed
When the Glide starts, VoiceResponder correctly says "...left supercruise in the vicinity of ..."
When the Glide completes, Environment contains "Supercruise"

## Investigation
Tested with the following VA call.  The "Else If Text Compare" test shows by writing to the VA console that the environment is incorrectly set, given the that the ship is actually at a planet and preparing to touch down.

```
Phrase: "Lower Landing Gear"

Write '[Blue] Lower landing gear' to log
Begin Text Compare : [Vehicle] Does Not Equal 'Ship'
    Set Text [Script] to 'Your {TXT:Vehicle} does not have a landing gear.'
    Execute external plugin, 'EDDI 3.0.1-rc2'
Else
    Begin Condition : [Status docked] Equals True OR [Status landing gear down] Equals True OR [Status landed] Equals True
        Execute command, '((Cannot Comply))' (and wait until it completes)
        Set Text [Script] to '[It;The gear] is [already;] down.'
        Execute external plugin, 'EDDI 3.0.1-rc2'
    Else If Text Compare : [Environment] Does Not Equal 'Normal space'
        Write '[Blue] {TXT:Environment}' to log
        Execute command, '((Cannot Comply))' (and wait until it completes)
        Set Text [Script] to '[We are not in normal space;We are going to fast;We cannot land out here;The frameshift drive is still active]'
        Execute external plugin, 'EDDI 3.0.1-rc2'
    Else
        Set small int (condition) [x] value as random from 0 to 1
        Begin Small Integer Compare : [x] Equals 0
            Execute command, '((Acknowledge))' (and wait until it completes)
        Else
            Set Text [Script] to '[lowering;dropping;deploying] [landing;] gear'
            Execute external plugin, 'EDDI 3.0.1-rc2'
        End Condition
        Execute command, '((LandingGear))' (and wait until it completes)
    End Condition
End Condition

```

**Comment by Corysia** [comment]
I found this in 3.0.1-b4, and it was still present in rc1.  I haven't tested with rc2, but the changelog doesn't show anything related to this.

**Comment by Hoodathunk** [comment]
Took a brief look at the code and can't see any reason for this behavior, but we'll investigate.

Regardless, I use 'Status supercruise' in my VA 'Landing Gear' command and it works fine.

**Comment by Hoodathunk** [comment]
OK, I think I know what's going on and have a hotfix in the works.

**Comment by richardbuckle** [comment]
PR #711. Fixed in 3.0.1-rc3.
